cli/src/main.rs

```diff
@@ -0,0 +1,112 @@
+use clap::{Parser, Subcommand};
+
+// mod build;
+// mod deps;
+// mod export;
+mod init;
+// mod prepare;
+// mod test;
+// mod update;
+// mod utils;
+
+/// CLI for multi-platform project management
+#[derive(Parser, Debug)]
+#[command(author, version, about, long_about = None)]
+struct Cli {
+    #[command(subcommand)]
+    command: Commands,
+}
+
+#[derive(Subcommand, Debug)]
+enum Commands {
+    // /// Install required dependencies
+    // Deps {},
+    /// Initializes a new project
+    Init {
+        #[arg(
+            long,
+            help = "Specify the adapter to use (e.g., 'circom', 'halo2' or 'circom,halo2')."
+        )]
+        adapter: Option<String>,
+        #[arg(long)]
+        project_name: Option<String>,
+    },
+    // /// Prepare and build circuit artifacts
```

Comment:
What are these comments for? Can they be removed?

Reply:
[132638c](https://github.com/zkmopro/mopro/pull/235/commits/132638c70c3e17db08632bd62a1933efe05006b7)